p4sdk/images/Dockerfile

```diff
@@ -1,27 +1,28 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright (c) 2024 Intel Corporation
-FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b as builder
-ARG TARGETOS
-ARG TARGETARCH
+
+FROM registry.access.redhat.com/ubi9/ubi:9.3-1610
 
 ENV HTTP_PROXY $http_proxy
 ENV HTTPS_PROXY $https_proxy
 
-RUN apk add --no-cache --virtual tar
 COPY . /src
 WORKDIR /src
-RUN mkdir -pv /opt/p4 && tar -zxvf p4.tar.gz -C /opt/p4 && rm -f p4.tar.gz
+RUN mkdir -pv /opt/ && tar -zxvf p4.tar.gz -C /opt/ && rm -f p4.tar.gz
+RUN mkdir -p /usr/share/target_sys/
+RUN cp /opt/p4/p4sde/share/target_sys/zlog-cfg /usr/share/target_sys/zlog-cfg
+RUN mkdir -p /usr/share/stratum/es2k/
+RUN cp /opt/p4/p4-cp-nws/share/stratum/es2k/es2k_port_config.pb.txt /usr/share/stratum/es2k/es2k_port_config.pb.txt
 
-FROM --platform=linux/arm64 alpine:3@sha256:c5b1261d6d3e43071626931fc004f70149baeba2c8ec672bd4f27761f8e1ad6b
-RUN apk add --no-cache --virtual openssl libressl-dev ncurses-libs libstdc++ gawk gettext
-COPY --from=builder /opt/p4 /opt/
 COPY p4 /root/linux_networking
 COPY images/entrypoint.sh /
 COPY images/es2k_skip_p4.conf.template /
+RUN dnf -y install kmod-28-9.el9.aarch64 gettext-0.21-8.el9.aarch64 python3-pip-21.2.3-7.el9_3.1.noarch pciutils-3.7.0-5.el9.aarch64 
+RUN dnf -y install libconfig-1.7.2-9.el9.aarch64 libnl3-3.7.0-1.el9.aarch64 libedit-3.1-38.20210216cvs.el9.aarch64 libatomic-11.4.1-2.1.el9.aarch64
+RUN dnf clean all
```

Comment:
I think the hadolint check above is looking for ` && dnf clean all` appended at the same line of `dnf -y install` command

Reply:
Done